Resources/public/scss/modules/content-tree/_list.scss

```diff
@@ -3,3 +3,10 @@
     padding: 0 0 0 calculateRem(12px);
     list-style: none;
 }
+
+.m-tree__scrollable-wrapper {
+    > .c-list {
+        display: table;
```

Comment:
I also tried solutions with 
- `display: inline-block`
- `display: flex; align-items: stretch;` on `.m-tree__scrollable-wrapper`
but they didn't work. I know that _table layouts_ are taboos in CSS world but this solutions seems to work perfectly.

If you have better idea @dew326 I will gladly hear it. 🙂

Reply:
Another this is, do we want to move this to `_main.scss` or leave it here? I added it here because it styles _c-list_.

Reply:
I think adding hover state on `c-list-item__link` should solve the problem.

Reply:
It's interesting idea, but it won't work with shorter items - we want all highlights to stretch to the right side of the container. Problematic example:

![screen shot 2019-03-06 at 14 21 32](https://user-images.githubusercontent.com/10233057/53884149-2b85e280-401b-11e9-9403-f6be137e47f3.png)

![screen shot 2019-03-06 at 14 18 25](https://user-images.githubusercontent.com/10233057/53884019-e3ff5680-401a-11e9-8057-561750772f28.png)